packages/node-cli/src/SdkExecutor.js

```diff
@@ -59,6 +59,16 @@ module.exports = class SdkExecutor {
 		});
 	}
 
+	// Taken from https://medium.com/@almenon214/killing-processes-with-node-772ffdd19aad
+	_killChildProcessAndItsChildren() {
```

Comment:
I'm not sure about the async of this function. Maybe need to add await's somewhere?

Reply:
good point - I don't think we actually need to wait for anything to finish but we may want to have a `catch` handler on `process.kill` because it could throw an error in some cases (for example, if the process already finished) and without a `catch` it will be an unhandled promise rejection.
(not sure there is much we can actually do with that error, but I guess we could at least log it...)